packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -98,6 +98,22 @@ class _FakeClosedCaptionFile extends ClosedCaptionFile {
 }
 
 void main() {
+  void verifyPlayingWhenAppLifecyclePaused(
+    VideoPlayerController controller, {
+    required bool isObserving,
+  }) {
+    final wasPlayingBeforePause = controller.value.isPlaying;
+    WidgetsBinding.instance!
```

Comment:
You'll need to add [`_ambiguate`](https://github.com/flutter/plugins/blob/1df2543c78d034cc82cc9ac0f63865f8dc8bf31e/packages/video_player/video_player/lib/video_player.dart#L1042-L1047) to this file and call it on these lines to fix the analyzer issue on master.